codeship-services.yml

```diff
@@ -12,8 +12,6 @@ web:
       - ./common.env
     links:
         - db
```

Comment:
Is now a good time to replace `links` with `depends_on` since `links` is deprecated?

Reply:
I don't think I'll take the time at the moment, but that would be good to do sometime soon. PRs welcome. 😉